apps/storybook/src/Slider.stories.tsx

```diff
@@ -65,10 +82,19 @@ MultiThumbsAllowCrossing.args = {
 };
 
 export const WithCustomThumb: Story<SliderProps> = (args) => {
-  return <Slider {...args} />;
+  const thumbProps = args.thumbProps();
+  thumbProps.style = {
+    ...thumbProps.style,
+    ...(args.orientation === 'horizontal' // TODO: Is there a way to get the args inside WithCustomThumb.args? Because just to gets args, I have to move those lines of code from WithCustomThumb.args to WithCustomThumb
+      ? { transform: 'translateX(-19.2px)', top: 0 }
+      : { transform: 'translate(-25%, 50%)' }),
+  };
+  args.thumbProps = () => thumbProps;
+  return sliderWrapper(<Slider {...args} />, args);
 };
 
 WithCustomThumb.args = {
+  // TODO: Should we remove it from the function since it cannot be edited in storybook?
   thumbProps: () => {
     return {
       style: {
```

Comment:
Just to confirm, with the way the code was, `thumbProps` cannot be edited from Storybook. It just This is because `thumbProps` is a function that returns an object and is not the object itself. Is that okay? Or should I unwrap `thumbProps` to an object and then rewrap it into a function before passing the args into the slider?

![image](https://user-images.githubusercontent.com/45748283/180845921-4c0e9e72-b5bd-40cd-9764-72ed8cea2097.png)

Example code:
```javascript
let thumbProps = args.thumbProps;
  thumbProps = {
    ...thumbProps,
    style: {
      ...thumbProps.style,
      ...(args.orientation === 'horizontal'
        ? { transform: 'translateX(-19.2px)', top: 0 }
        : { transform: 'translate(-25%, 50%)' }),
    },
  };
  args.thumbProps = () => thumbProps; // Rewrap the object into a method
```

Reply:
**[DONE]** Thanks